src/tldr.c

```diff
@@ -82,12 +82,13 @@ int main(int argc, char **argv) {
             break;
 
         case 'v':
-            verbose_flag = 1;
+            version_flag = 1;
             break;
 
         case '?':
             /* do not set the help flag, only show getopt error */
             /* help_flag = 1; */
+            return EXIT_FAILURE;
```

Comment:
See below comment about exiting with code `0` instead of `1` being preferable when printing help.